doc/sphinxext/ipython_sphinxext/ipython_directive.py

```diff
@@ -799,13 +799,19 @@ def setup(self):
             # Store IPython directive to enable better error messages
             self.shell.directive = self
 
-        # reset the execution count if we haven't processed this doc
-        #NOTE: this may be borked if there are multiple seen_doc tmp files
-        #check time stamp?
-        if not self.state.document.current_source in self.seen_docs:
+        """
+        reset the execution count if we haven't processed this doc
+        NOTE: this may be borked if there are multiple seen_doc tmp files
+        check time stamp?
+        """
```

Comment:
can you leave this as `#` comments? (that's how multiline comments in the codebase are done `"""` is used for docstrings)